iron-scroll-manager.html

```diff
@@ -224,8 +237,14 @@
     _boundScrollHandler: undefined,
 
     _lockScrollInteractions: function() {
-      this._boundScrollHandler =
-          this._boundScrollHandler || this._scrollInteractionHandler.bind(this);
+      // workaround for modulizer not handling bind properly
+      if (!this._boundScrollHandler) {
```

Comment:
This looks quite ugly :x
two options to explore:
1. Use only `_scrollInteractionHandler` and set at the bottom of the file 
```js
Polymer.IronScrollManager._scrollInteractionHandler = 
  Polymer.IronScrollManager._scrollInteractionHandler.bind(Polymer.IronScrollManager);
```
2. avoid using `.bind()` and rename `_scrollInteractionHandler` to `handleEvent`, so that we can add/remove listeners to `this` - https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

Reply:
1 will not work with modulizer because modulizer removes the entire `Polymer.IronScrollManager` definition and will keep it inside that bind (thus throwing Polymer is undefined errors)

2, I don't think I fully understand your explanation, but the issue is that modulizer would remove the entire `Polymer.IronScrollManager` namespace and simply export the functions. Since, it removes the namespace and it's just naked functions in a module, then `this` is undefined.

I think a good 3 would be to make this into a class instead of a namespace, but I believe that will break IE in Polymer 1.

Reply:
you can add an event listener like this
```js
document.addEventListener('scroll', this);
```
^ when scroll is fired, `this.handleEvent(event)` is invoked.
So the suggestion is to completely remove `_boundScrollHandler` and rename `_scrollInteractionHandler` to `handleEvent`